server/lists/adapters/workstacks.js

```diff
@@ -27,7 +27,7 @@ const defaultCaseSort = (a, b) => {
 };
 const byLabel = (a, b) => a.label.localeCompare(b.label);
 const isUnallocated = user => user === null;
-const isOverdue = (configuration, deadline) => configuration.deadlinesEnabled && deadline && new Date(deadline) < Date.now();
+const isOverdue = (configuration, deadline) => configuration.deadlinesEnabled && deadline && new Date(deadline).setHours(23, 59, 59, 999) < Date.now();
```

Comment:
This is an interesting approach. Would it be better to check if it's after 00:00:00 tomorrow, rather than 23:59:59 today?

The line's also getting a bit long -- I'd add a couple of line-breaks.

Reply:
We'd have to do something like this: 
https://stackoverflow.com/a/563442

But that would require more computation, this solution is simpler. 

Will add more line breaks, to simplify the view.

Reply:
This has been rectified 👍🏼